### PR TITLE
Dayr as suryan

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Completed

### CODE REVIEW

File: DayrSuryan/DSEthiop7.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <locus from="1r" to="51r" cert="medium"/>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop7.xml]: cert="medium" does actually not give very meaningful information...maybe you could phrase the uncertainty better in a note?

File: DayrSuryan/DSEthiop7.xml [hunk 2]
```diff
@@ -75,82 +75,82 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                     <incipit>ዘሊቃውንት᎓ ልጦን᎓ ዘስኑይ᎓ ወካዕበ᎓ ናስተበቍዕ᎓ ዘኵሎ᎓ ይእኅዝ᎓ እ<hi rend="ligature">ግዚ</hi>አብሔር᎓
+                     <incipit xml:lang="gez">ዘሊቃውንት᎓ ልጦን᎓ ዘስኑይ᎓ ወካዕበ᎓ ናስተበቍዕ᎓ ዘኵሎ᎓ ይእኅዝ᎓ እ<hi rend="ligature">ግዚ</hi>አብሔር᎓
                    አብ᎓ ለእግዚእነ᎓ ወመድኃኒነ᎓ ኢየሱስ᎓ ክርስቶስ᎓ እንዘ᎓ ነአኵት᎓ እስመ᎓ ዓቀበነ᎓ በኑኃ᎓ ሌሊት᎓ ወአብጽሐነ᎓
                  እምጽልመት᎓ ውስተ᎓ ብርሃን᎓ እሙስና፨ ውስተ᎓ ዘኢይማስን᎓ እምእያምሮ᎓ ውስተ᎓ አእምሮ᎓ አማን፨</incipit>
-                 <explicit>ወሚጥ᎓ መዓተከ᎓ እምኔነ᎓ ሃሌ᎓ ሉያ᎓ እግዚኦ፨</explicit>
-                     <note>Litanies for the day of the week. The dext is divided into the portions for the days of the week (after the
-                      part for Sunday, <foreign xml:lang="gez">ዘሰሉስ</foreign>,
+                 <explicit xml:lang="gez">ወሚጥ᎓ መዓተከ᎓ እምኔነ᎓ ሃሌ᎓ ሉያ᎓ እግዚኦ፨</explicit>
+                     <note>Litanies for the day of the week. The text is divided into the portions for the days of the week (after the
+                      part for Sunday: <foreign xml:lang="gez">ዘሰሉስ</foreign>,
```
Comment (thea-m) [inline DayrSuryan/DSEthiop7.xml]: for the days, maybe?

File: DayrSuryan/DSEthiop7.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop7.xml]: solved

File: DayrSuryan/DSEthiop7.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop7.xml]: yes, corrected.

**Pull request merged**